java-manta-client/src/test/java/com/joyent/manta/client/crypto/EncryptingEntityTest.java

```diff
@@ -25,6 +27,20 @@
 
 @Test
 public class EncryptingEntityTest {
+    /* Constructor Tests */
+
+    @Test(expectedExceptions = MantaClientEncryptionException.class)
+    public void throwsWithTooLargeContentLength() {
+        SupportedCipherDetails cipherDetails = AesGcmCipherDetails.INSTANCE_128_BIT;
+        byte[] keyBytes = SecretKeyUtils.generate(cipherDetails).getEncoded();
+
+        SecretKey key = SecretKeyUtils.loadKey(ByteUtils.subArray(keyBytes, 2), cipherDetails);
```

Comment:
Use the method java.util.Arrays.copyOfRange() instead of the library from Bouncy Castle.